### PR TITLE
Update contributing documentation

### PR DESCRIPTION
**Pull request by redsummernight**
## Issue

None.

## Purpose

- Add a reminder in the pull request checklist to keep <= 5 pull requests open
  at the same time.
- Remove unused issue template.
- Restructure CONTRIBUTING.md.
- Add SECURITY.md.

[skip ci]

## Testing Instructions

Please proof-read. You can find a preview at https://github.com/redsummernight/otwarchive/tree/contributing.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Pull Request Checklist
 
-* [ ] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
-* [ ] Have you read through the [contributor guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
-* [ ] Have you added tests for any changed functionality?
-* [ ] Have you added the [JIRA](https://otwarchive.atlassian.net) issue number as the *first* thing in your pull request title (eg: `AO3-1234 Fix thing`)
-* [ ] Have you updated or commented on the JIRA issue with the information below?
+* [ ] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
+* [ ] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
+* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
```
Comment (sarken) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Nice 👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+* [ ] Have you added the [JIRA](https://otwarchive.atlassian.net) issue number
```
Comment (sarken) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: Should we do "Jira" now that they've changed their capitalization?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -16,19 +18,23 @@ What does this PR do?
...
-How can the Archive's QA team verify that this is working as you intended? (If you have access, please copy this into the JIRA ticket for them!)
+How can the Archive's QA team verify that this is working as you intended?
+
+(If you have a JIRA account with access, please update or comment on the issue
+with any new or missing testing instructions.)
 
 ## References
 
-Are there any other relevant issues / PRs / mailing lists discussions related to this?
+Are there other relevant issues / pull requests / mailing lists discussions?
```
Comment (sarken) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: Can we get rid of the spaces around the slashes? Also, it should probably be "mailing list discussions" -- no `s` on lists.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,57 +1,64 @@
...
 
-## Contributing code
 
-### Report a bug
+## Reporting security issues
 
-We have a [list of known issues](https://archiveofourown.org/known_issues).
-If you have a problem that you don't see on that list, please send a bug report
-via the [Support and Feedback Form](https://archiveofourown.org/support).
+Please see [SECURITY.md](https://github.com/otwcode/otwarchive/blob/master/SECURITY.md).
```
Comment (sarken) [inline CONTRIBUTING.md:16]: "Refer to" instead of "see"?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,57 +1,64 @@
...
 
-### Pick an issue to work on
 
-You will find our [list of approved issues on JIRA](https://otwarchive.atlassian.net/projects/AO3/issues).
+## Suggesting new features
```
Comment (sarken) [inline CONTRIBUTING.md:19]: 🍷

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Reporting a security issue
+
+Please email us directly at [otw-coders@transformativeworks.org](mailto:otw-coders@transformativeworks.org)
+with details and reproduction steps. We will get back to you as soon as possible,
+and we may ask for additional information or guidance.
+
+Please avoid testing for security issues on the Archive of Our Own itself,
+as you risk disrupting other users and violating the [Terms of Service](https://archiveofourown.org/tos).
```
Comment (sarken) [inline SECURITY.md:8]: Oh, should we link directly to the relevant section? https://archiveofourown.org/tos#IV.C.

File: CONTRIBUTING.md (hunk 5 above)
Comment (redsummernight) [reply CONTRIBUTING.md:19]: Should we move this section lower?

Comment (sarken) [reply CONTRIBUTING.md:19]: Hmm. Yeah, sure, let's put it after "Updating documentation."

**Comment by bingeling** [comment]
All looks good to me!

**Pull request merged**